### DISCUSSION TITLE
Cron schedule no longer working on 2.0b8

### DISCUSSION THREAD

**Discussion by sdittmar** [PrefectHQ/prefect#5983] [Q&A] [answered]
`schedule=CronSchedule(cron="30 8 * * MON-FRI", timezone="Europe/Zurich") `

used to work until the latest update. Any idea why Prefect returns this error now:

```
File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for CronSchedule
cron
  Invalid cron string: "30 8 * * MON-FRI" (type=value_error)
```

**Comment by sdittmar** [comment, marked as answer]
Solution:

`schedule=CronSchedule(cron="30 8 * * 1-5", timezone="Europe/Zurich")`

**Comment by znicholasbrown** [comment]
Hi @sdittmar - thanks for raising; this looks like an issue on our end where we were incorrectly validating cron expressions. We'll get a patch out for this as soon as possible!
